### PR TITLE
Created schema for reset token

### PR DESCRIPTION
**Pull request by crowbardispensingcore**

### CODE REVIEW

File: api/main_endpoints/models/password-reset.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PasswordReset = new Schema(
+  {
+    email: {
```
Comment (allen-tran) [inline api/main_endpoints/models/password-reset.js:6]: two things:
- [ ] the definition of the schema should follow the naming conventions of the other schemas
- [ ] the file name should also follow the naming convention of the other files in the models folder

File: api/main_endpoints/models/password-reset.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
```
Comment (allen-tran) [inline api/main_endpoints/models/password-reset.js:1]: this only change should be this single file, you can undo the other changes by resetting it with dev with `git checkout origin/dev filename`

File: api/main_endpoints/models/password-reset.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      required: true
+    },
+    createdAt: {
+      type: Date,
+      default: Date.now,
+      required: true,
+      index: {  expires: '1h' }
+    }
+  }
+);
```
Comment (allen-tran) [inline api/main_endpoints/models/password-reset.js:21]: this is solid tho and ily

**Review by allen-tran — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -72,6 +72,7 @@
     "react-scripts": "^4.0.3",
     "reactstrap": "^8.10.0",
     "serve": "^11.3.2",
+    "server": "^1.0.37",
```
Comment (neelp03) [inline package.json:75]: package.json should remain the same as dev right?

**Review by neelp03 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**